apps/src/sites/code.org/pages/public/educate/weblab-test.js

```diff
@@ -0,0 +1,11 @@
+import $ from 'jquery';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import WebLabTest from '@cdo/apps/templates/testPages/WebLabTest.jsx';
```

Comment:
nit: just noticed that naming this `*Test` looks somewhat like the way we name our tests.  No need to change right now, but I wonder if we could have named it `WebLabCheck` or something like that?

Reply:
I was also wondering if we could just do `templates/testPages/WebLab.jsx`. I'm not sure how much we rely on descriptive component names vs. having the file structure/directory do some of that description for us.

Reply:
I'd personally go for something a bit more descriptive than just `WebLab`, since modern editors (and browser source file searches) let us jump direct to filenames without knowing the directory they are in.